src/pages/dashboard/_SmallMultiplesContainer.js

```diff
@@ -105,11 +105,12 @@ const SmallMultiplesContainer = () => {
   `)
 
   const data = groupAndSortStateDaily(query)
-  const secondMaxTotal = useMemo(() => {
-    if (!data[1] && !data[0]) return
-    // eslint-disable-next-line consistent-return
-    return max(data[1].values, d => calculateTotal(d))
-  }, [data.length])
+
+  const maxStateTests = useMemo(() => {
+    return max(data, stateData => {
+      return max(stateData.values, d => calculateTotal(d))
+    })
+  })
```

Comment:
@jeremiak I revised this logic determining the max Y scaling. Unsure why code was using the first two only originally, but maybe I'm missing some context.

Reply:
we originally had NY going off the graph to demonstrate how bad things have gotten there.